purchase_reception_status/views/purchase_order.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2020 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="before">
+                <field
+                    name="reception_status"
+                    invisible="state not in ['purchase', 'done']"
+                />
+                <field
+                    name="force_received"
+                    groups="purchase.group_purchase_manager"
+                    invisible="state not in ['purchase', 'done']"
+                    readonly="state != 'done'"
+                />
+            </field>
+        </field>
+    </record>
+    <record id="purchase_order_tree" model="ir.ui.view">
```

Comment:
Perhaps a good improvement would be to have the field also enabled on the default RFQ tree view: `purchase.purchase_order_kpis_tree`. WDYT? 🙂

Reply:
Disregard this comment, I believe that it should be kept as is, the `reception_status` is only shown when in the `purchase` state already.